code/game/objects/items.dm

```diff
@@ -602,7 +612,7 @@ var/global/image/fire_overlay = image("icon" = 'icons/effects/fire.dmi', "icon_s
 			var/index = blood_splatter_index()
 			var/icon/blood_splatter_icon = blood_splatter_icons[index]
 			if(blood_splatter_icon)
-				cut_overlay(blood_splatter_icon)
+				cut_overlay(blood_splatter_icon)
```

Comment:
**_Please do not manually change the overlays_**

Use `/proc/cut_overlays()` and `/proc/add_overlay(image, priority = 0)` so that overlay changes do not cause any lag